doc/manual.rst

```diff
@@ -4459,6 +4459,28 @@ Funding Rate History Structure
        datetime: "2022-01-23T16:00:00.000Z"
    }
 
+Open Interest
+---------------------
+
+ *contract only*
+
+Use the ``fetchOpenInterest`` method to get a the open interest for a symbol from the exchange.
+
+.. code-block:: JavaScript
+
+   fetchOpenInterest (symbol, params = {})
+
+Parameters
+
+
+ * **symbol** (String) Unified CCXT market symbol (e.g. ``"BTC/USDT:USDT"``\ )
+ * **params** (Dictionary) Extra parameters specific to the exchange API endpoint (e.g. ``{"endTime": 1645807945000}``\ )
+
+Returns
+
+
+ * A dictionary of :ref:`open interest structures <open interest structure>`
+
```

Comment:
I think manual.rst is a compiled file. When I add a section to the manual, I add it inside `wiki/Manual.md`

Also add the method name to https://github.com/ccxt/ccxt/blob/master/wiki/Manual.md#unified-api